src/openai.rs

```diff
@@ -120,46 +119,13 @@ impl OpenAI {
         );
 
         //Check how many tokens are required for prompt
-        let bpe = get_tokenizer(&self.model)?;
+        let bpe = get_tokenizer_old(&self.model)?;
         let prompt_tokens = bpe.encode_with_special_tokens(&full_prompt).len();
 
         //Assuming another 5% overhead for json formatting
         Ok((prompt_tokens as f64 * 1.05) as usize)
     }
 
-    /*
-     * This function leverages OpenAI API to perform any query as per the provided body.
-     *
-     * It returns a String the Response object that needs to be parsed based on the self.model.
-     */
-    async fn call_openai_api(&self, body: &serde_json::Value) -> Result<String> {
```

Comment:
This function was specific to the model so I moved it from the openai/completions struct to the model implementation.